tests/lean/run/smt2_p_implies_p.lean

```diff
@@ -0,0 +1,7 @@
+import tools.smt2
+
+example (P : Prop) : P → P :=
+begin
+    intros,
+    z3 $ "/tmp/P.smt2"
```

Comment:
This was left overs from testing, got distracted yesterday afternoon by some other things and haven't had a chance to come back yet.